core/templates/dev/head/pages/collection_editor/settings_tab/collection_settings_tab_directive.html

```diff
@@ -6,4 +6,10 @@
       <collection-details-editor></collection-details-editor>
     </md-card>
   </div>
+  <div class="oppia-page-cards-container">
+    <md-card class="oppia-page-card oppia-long-text">
+      <h2>Collection Permissions</h2>
+      <collection-permissions-editor></collection-permissions-editor>
```

Comment:
Maybe fold the md-card stuff into the directive? Then the component 'represents' the collection permissions card, which is quite nice conceptually.